### PR TITLE
Display version for groups in application list. 

### LINKED ISSUE #312: Display version(s) for groups in application list
Right now, in spring-boot-admin-server-ui/modules/applications/views/applications-list.html, only name and status is displayed for each grouped application. For us, it would be very useful to also have the version there. My suggestion:

If all instances have the same version, display that version. If not, display some text, for example: "Versions differ", or something like that.

The objective is to get a quick overview of what is deployed.

What do you think?

### PR DESCRIPTION
**Pull request by anderius**
Resolves #312.

### CODE REVIEW

**Comment by anderius** [comment]
The build fails even before I make my changes, maybe because I am on Windows? So I have not been able to test this.

If I disable the npm test part of the test phase, I am able to build, but I get the following error when acessing the ui:

`Uncaught Error: [$injector:modulerr] Failed to instantiate module sba-about due to:
Error: [$injector:modulerr] Failed to instantiate module sba-core due to:
Error: [$injector:nomod] Module 'sba-core' is not available! You either misspelled the module name or forgot to load it. If registering a module ensure that you specify the dependencies as the second argument.`

File: spring-boot-admin-server-ui/modules/applications/services/applicationGroups.js [hunk 1]
```diff
@@ -31,6 +31,14 @@ module.exports = function () {
       return 'UNKNOWN';
     };
 
+    var getGroupVersion = function (versionCounter) {
+      if (versionCounter.length === 1) {
+        return versionCounter[0];
+      } else {
+        return '- multiple versions -';
```
Comment (anderius) [inline spring-boot-admin-server-ui/modules/applications/services/applicationGroups.js]: Not sure how best to display this...

**Comment by joshiste** [comment]
>The build fails even before I make my changes, maybe because I am on Windows? So I have not been able to test this.

Either you do a full build and start one of the samples.
Or you start the sample and do a `npm run dev-server` in the spring-boot-admin-server-ui folder should work...

**Comment by joshiste** [comment]
merged with dca3e2903d1051a9fedb4091f9ab60c3110e3313.

**Comment by joshiste** [comment]
I needed to modify the code slightly to make things work.
If multiple versions are found the lowest version (by lexical compare) gets displayed with `, ...` appended.

**Comment by anderius** [comment]
Excellent, that way to display multiple versions was much, much better! Thank you, my team will look forward to the next release!

**Pull request closed without merging**